src/codegen/llvm.zig

```diff
@@ -11145,6 +11145,7 @@ fn isScalar(ty: Type) bool {
 fn backendSupportsF80(target: std.Target) bool {
     return switch (target.cpu.arch) {
         .x86_64, .x86 => !std.Target.x86.featureSetHas(target.cpu.features, .soft_float),
+        .aarch64 => std.Target.aarch64.featureSetHas(target.cpu.features, .fp_armv8),
```

Comment:
```suggestion
```
Aarch64 does not support f80.

Reply:
Thanks a lot, it must be this that's making the tests not pass. I could not see it.